iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/helpers/IotHubServicesCommon.java

```diff
@@ -171,61 +171,6 @@ public static void sendMessagesExpectingConnectionStatusChangeUpdate(InternalCli
         Assert.assertTrue(buildExceptionMessage(protocol + ", " + authType + ": Expected connection status update to occur: " + expectedStatus, client), actualStatusUpdatesContainsStatus(actualStatusUpdates, expectedStatus));
     }
 
-    /**
-     * Send some messages that wait for callbacks to signify that the SAS token in the client config has expired.
-     *
-     * @param client the client to send the messages from
-     * @param protocol the protocol the client is using
-     * @param numberOfMessages the number of messages to send
-     * @param breathPeriod time to sleep between checks of message callback arrival
-     * @param timeoutMilliseconds time to wait for any message to have its callback fired before the test times out
-     * @param authType the authentication type used is this test
-     */
-    public static void sendMessagesExpectingSASTokenExpiration(DeviceClient client,
```

Comment:
This logic was tied to a test of a once deprecated and now removed connection status callback. We still have a connection status change callback, though.